hooks/pre-commit

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from future import standard_library
+standard_library.install_aliases()
+
+import commands
+import os
+import sys
+import urllib.request, urllib.parse, urllib.error
+from distutils.version import LooseVersion
+import re
+
+pattern = re.compile(r'[."]version["\s":=]+(\d+\.\d+.\d+)')
+version_dict = {}
+versionfile = urllib.request.urlopen(
+    "https://raw.githubusercontent.com/weecology/retriever/master/version.txt")
+versionfile.readline()
+
+for lines in versionfile:
+    key_values = lines.split(",")
+    version_dict[key_values[0]] = key_values[1]
+
+
+def get_script_version(file_name):
+    file_name = os.path.split(file_name)[1]
+    return LooseVersion(version_dict[file_name])
+
+
+# check if script directory changes
+remotes = subprocess.check_output("git remote -v")
+if("upstream    https://github.com/weecology/retriever.git" in remotes):
+    status, output = commands.getstatusoutput("git diff --name-only upstream/master scripts")
+else:
+    print("Please add weecology/retriever as a remote. \n git remote add upstream https://github.com/weecology/retriever.git")
+    exit(1)
+# get list of changed scripts
+scripts_changed = output.split("\n")
+
+# get the changes staged for the next commit relative to HEAD
```

Comment:
Remove # Still needs testing on python3 and python2
Rename version_files to staged_scripts
```
scripts_changed = output.split("\n")

# get the changes staged for the next commit relative to HEAD
staged_scripts = subprocess.check_output("git diff --cached --name-only", shell = True).decode("UTF-8")

versions_not_changed =[]  # if script changes we expect to change the version
versions_updated = True
for files in scripts_changed:
    if files in staged_scripts:
```